caffe2/python/examples/resnet50_trainer.py

```diff
@@ -338,12 +365,14 @@ def test_input_fn(model):
                 test_reader,
                 batch_size=batch_per_device,
                 img_size=args.image_size,
+                dtype=args.dtype,
             )
 
         data_parallel_model.Parallelize(
             test_model,
             input_builder_fun=test_input_fn,
             forward_pass_builder_fun=create_resnet50_model_ops,
+            post_sync_builder_fun=add_post_sync_ops,
```

Comment:
It doesn't cause an error. I confess I'm not entirely sure what this step accomplishes - @slayton58 wrote this part.